<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NostoForm</class>
 <widget class="QWidget" name="NostoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nosto</string>
  </property>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Valitse Summa </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVahvista">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 153, 11);</string>
   </property>
   <property name="text">
    <string>Vahvista</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Marker Felt</family>
     <pointsize>46</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bank Automat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20euroa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>20 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn60euroa">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>60 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn40euroa">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>40 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn80euroa">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>80 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn100euroa">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>100 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn50euroa">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>50 €</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMuuSumma">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMuu">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Muu Summa :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNostoStatus">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
